components/modal/withdraw-modal.tsx

```diff
@@ -1,67 +1,41 @@
-import React, { useMemo, useState } from 'react'
-import { formatUnits, isAddressEqual, parseUnits } from 'viem'
-import { useQuery } from 'wagmi'
+import React from 'react'
+import { formatUnits } from 'viem'
 import { min } from 'hardhat/internal/util/bigint'
```

Comment:
p3; `hardhat bigint min` seems to be compatible with storybook

Reply:
p2; How about pass `disable` parameter from Container?